estimagic/differentiation/hessian.py

```diff
@@ -0,0 +1,70 @@
+import numdifftools as nd
+import numpy as np
+import pandas as pd
+
+from estimagic.differentiation.first_order_auxiliary import central
+
+
+def hessian(
+    func,
+    params_sr,
+    method="central",
+    extrapolation="richardson",
+    func_args=None,
+    func_kwargs=None,
+):
+    """
+    Calculate the hessian of *func*.
+
+    Args:
+        func (function): A function that maps params_sr into a float.
+        params_sr (Series): see :ref:`parmas_df`
+        method (string): The method for the computation of the derivative. Default is
+                         central as it gives the highest accuracy.
+        extrapolation (string): This variable allows to specify the use of the
+                                richardson extrapolation.
+        func_args (list): additional positional arguments for func.
+        func_kwargs (dict): additional positional arguments for func.
+
+    Returns:
+        DataFrame: The index and columns are the index of params_sr.
+
+    """
+    if method != "central":
+        raise ValueError("The given method is not supported.")
+    # set default arguments
+    func_args = [] if func_args is None else func_args
+    func_kwargs = {} if func_kwargs is None else func_kwargs
+    if extrapolation == "richardson":
```

Comment:
Extrapollation should be True/False, not a string. A string would only make sense if we support several methods. Do other methods exist?

Reply:
Changed from string to bool.